spec/registry.md

```diff
@@ -833,3 +818,25 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 > The form of these serializations is known and is a de facto standard.
 > Support for these extensions is expected to be required in the post-tech preview.
 > See: https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/
+
+### Date and Time Locale Options
+
+**_<dfn>Date/time locale options</dfn>_** override options set by the current locale
+or an implementation-defined date/time _operand_ value.
+
+The following options and their values are required to be available on
+the functions `:datetime`, `:date`, and `:time`:
+
+- `hour12`
+  - `true`
+  - `false`
```

Comment:
I thought we said that this option would _replace_ `hourCycle` in the `:time` and `:datetime` functions above. Having both seems bad.

It should also say that the default is locale-defined.

Reply:
This PR removes `hourCycle` from `:datetime`, and it was never available on `:time`.

The default being locale-defined is mentioned in the note below the list.